contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
+
+## Which trailers are to be added?
+
+- `Helped-by: Full Name <email@address>`
+   Added by regular users to particular commits within a PR. Applies to the commit it appears on. Means "this commit was somehow improved based on suggestions by @reviewer".
+- `Reviewed-by: Full Name <email@address>`
+   Added by maintainer to PR merge commit. Applies to whole PR. Means "these changes were carefully reviewed by @reviewer an he has stated conformity with them."
```

Comment:
> I just think it's a bit of a pity some people make an argument out of that. To me, offense is in the intention, not in the words you use.

In general I agree with this. I was just assuming, if we agree trailers are the way to go, this paragraph would be moved to `CONTRIBUTING.md` or similar (as is not really related to this tool), at which point it would be a "policy" document and not "personal communication" anymore, and a bit more care might be warranted. But not "big issue" or anything, just my opinion :)